lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -1420,17 +1503,17 @@ class NGPHistoryParser {
       .with_recommended({
         carb: wizardEvent.foodEstimate,
         correction: wizardEvent.iobAdjustment,
-        net: wizardEvent.bolusWizardEstimate,
+        net: wizardEvent.wizardEstimate,
       })
       .with_bolus(wizardBolus)
       .with_carbInput(wizardEvent.carbInput)
       .with_insulinOnBoard(wizardEvent.iob)
       .with_insulinCarbRatio(wizardEvent.carbRatio)
       .with_insulinSensitivity(wizardEvent.isf)
-      .with_bgTarget({
+      .with_bgTarget(wizardEvent.eventType === NGPHistoryEvent.EVENT_TYPE.BOLUS_WIZARD_ESTIMATE ? {
         low: wizardEvent.lowBgTarget,
         high: wizardEvent.highBgTarget,
-      })
+      } : undefined)
```

Comment:
I'd rather we wrap this in an if statement, e.g.,

```
if (wizardEvent.eventType === NGPHistoryEvent.EVENT_TYPE.BOLUS_WIZARD_ESTIMATE) {
  wizard.with_bgTarget({
    low: wizardEvent.lowBgTarget,
    high: wizardEvent.highBgTarget,
  })
}
```
Otherwise it will send up a `bgTarget : undefined` property, when it should not be sending anything.